### PR TITLE
Remove CUPS PPD API dependancy.

### PR DESCRIPTION
**Pull request by NJRoadfan**
Since the CUPS PPD API is going away in CUPS 3.0, we'll just generate our own.

This PR generates a very basic PPD from a printer's IPP attributes for papd to use to respond to printer information queries by the LaserWriter 8 driver.

### CODE REVIEW

**Comment by NJRoadfan** [comment]
Fixes #31.

File: etc/papd/print_cups.c [hunk 1]
```diff
@@ -205,8 +193,199 @@ cups_printername_ok(char *name)         /* I - Name of printer */
...
+	}
+
+	if ((http = cupsConnectDest(dest, CUPS_DEST_FLAGS_NONE, 30000, NULL, NULL, 0, NULL, NULL)) == NULL)
+	{
+		LOG(log_error, logtype_papd, "testdest: Unable to connect to destination \"%s\": %s\n", dest->name, cupsLastErrorString());
+		cupsFreeDests(num_dests,dests);
+		return (0);
+	}
+
+	sprintf(uri, "ipp://localhost/printers/%s", name);
```
Comment (rdmark) [inline etc/papd/print_cups.c:246]: What does this line do?

File: etc/papd/print_cups.c [hunk 2]
```diff
@@ -205,8 +193,199 @@ cups_printername_ok(char *name)         /* I - Name of printer */
...
+
+	cupsFilePuts(fp, "*PPD-Adobe: \"4.3\"\n");
+	cupsFilePuts(fp, "*FormatVersion: \"4.3\"\n");
+	cupsFilePrintf(fp, "*FileVersion: \"%d.%d\"\n", CUPS_VERSION_MAJOR, CUPS_VERSION_MINOR);
+	cupsFilePuts(fp, "*LanguageVersion: English\n");
+	cupsFilePuts(fp, "*LanguageEncoding: ISOLatin1\n");
+	cupsFilePuts(fp, "*PSVersion: \"(3010.000) 0\"\n");
+	cupsFilePuts(fp, "*LanguageLevel: \"3\"\n");
+	cupsFilePuts(fp, "*FileSystem: False\n");
+	cupsFilePuts(fp, "*PCFileName: \"ippeve.ppd\"\n");
```
Comment (rdmark) [inline etc/papd/print_cups.c:355]: Would be helpful for posterity to have a comment explaining why this hardcoding is happening...

And, does this mean we cannot print text in encodings other than ISO Latin? I'll test tomorrow with KanjiTalk! :)

**Review by rdmark — APPROVED** [review]

**Review by rdmark — APPROVED** [review]

File: etc/papd/print_cups.c (hunk 2 above)
Comment (NJRoadfan) [reply etc/papd/print_cups.c:355]: This is a standard PPD file header. Most of it isn't actually used by papd. The Language options are relevant only to the PPD file and printer drivers that would read them (basically, "This file is in English, parse as such"). You can print all the KanjiTalk you want.

**Pull request merged**